### PR TITLE
Split site into two.

### PR DESCRIPTION
**Pull request by lpil**
rainbow-head

- [x] Conference can be accessed from react.london
- [x] Community can be accessed from meetup.react.london
- [x] Old community & conference routes redirect to new URLs
- [x] New header Community
- [x] New header for Conference

### CODE REVIEW

File: shared/components/NavigationBar/index.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import React, { PropTypes } from 'react';
+import React from 'react';
 import { Link } from 'react-router';
 
-const NavigationBar = ({ page }) => (
-  <nav className={`NavigationBar block NavigationBar--${page}`}>
+const NavigationBar = () => (
+  <nav className={'NavigationBar block NavigationBar--Conference'}>
```
Comment (AndrewBestbier) [inline shared/components/NavigationBar/index.js]: What was the point of these changes?

File: shared/components/RedBadgerBanner/index.js [hunk 2]
```diff
@@ -2,19 +2,8 @@ import React from 'react';
 
 const RedBadgerBanner = () => (
   <section className="RedBadgerBanner block">
-    <div className="content space-between">
-      <object
-        className="RedBadgerBanner__logo"
-        data="/img/SVG/Badger.svg"
```
Comment (AndrewBestbier) [inline shared/components/RedBadgerBanner/index.js]: :(

Comment (lpil) [inline shared/components/RedBadgerBanner/index.js]: I know!!! :(

File: shared/components/NavigationBar/index.js (hunk 1 above)
Comment (lpil) [inline shared/components/NavigationBar/index.js]: There is only one navbar now, the conference one.

File: server/app.js [hunk 3]
```diff
@@ -33,6 +29,10 @@ if (process.env.NODE_ENV === 'production') {
   app.use(enforceHTTPS());
 }
 
+function isMeetupRequest(req) {
```
Comment (lpil) [inline server/app.js]: I don't like this- regular function definitions will get hoisted to the top, right?

Comment (lpil) [inline server/app.js]: Or does it? I dunno. How does hoisting work again? :(

**Pull request merged**